packages/skeleton-react/src/lib/components/Rating/Rating.test.tsx

```diff
@@ -1,204 +0,0 @@
-import { render, fireEvent } from '@testing-library/react';
```

Comment:
Make sure to reimplement the test spec file, even if it's just a single test case for now. I'd understand if the changes are dramatic enough to warrant us starting from scratch. But I want to ensure we have the test spec in place when we need it.

Reply:
What do we test? Since zag already tests all the logic. Ensuring class x is applied and such?

Reply:
* Barebones "does it work" with minimal props
* Anything Skeleton specific
* I'd recommend some light tests for style props like `base/classes`

Reply:
Thanks, I'll be sure to test this 👍